msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase1561151.java

```diff
@@ -0,0 +1,176 @@
+                        .build();
+
+                new AadPromptHandler(promptHandlerParameters)
+                        .handlePrompt(username, password);
+            }
+        }, TokenRequestTimeout.MEDIUM);
+
+        authResult.assertSuccess();
+
+        /*
+            Note that password reset doesn't take effect by ESTS at least user being logged in for 1 min.
+            Therefore we have a Thread.sleep after first successful token acquisition before resetting password.
+         */
+        Thread.sleep(TimeUnit.MINUTES.toMillis(2));
+        mLabClient.resetPassword(username);
+
+        TestContext.getTestContext().getTestDevice().getSettings().forwardDeviceTimeForOneDay();
+
+        // SILENT REQUEST - start a acquireTokenSilent request in MSAL
+        final IAccount account = msalSdk.getAccount(mActivity, getConfigFileResourceId(), username);
+        final MsalAuthTestParams silentParams = MsalAuthTestParams.builder()
+                .activity(mActivity)
+                .loginHint(username)
+                .authority(account.getAuthority())
+                .forceRefresh(true)
+                .scopes(Arrays.asList(mScopes))
+                .msalConfigResourceId(getConfigFileResourceId())
+                .build();
+
+
+        // get a token silently
+        final MsalAuthResult silentAuthResult = msalSdk.acquireTokenSilent(silentParams, TokenRequestTimeout.SILENT);
+        silentAuthResult.assertFailure();
+
+        // fetch token in an interactive request
+        final MsalAuthTestParams msalAuthTestParams2 = MsalAuthTestParams.builder()
+                .loginHint(username)
+                .activity(mActivity)
+                .scopes(Arrays.asList(mScopes))
+                .promptParameter(Prompt.WHEN_REQUIRED)
+                .msalConfigResourceId(getConfigFileResourceId())
+                .build();
+
+        final MsalAuthResult authResultPostPwdChange = msalSdk.acquireTokenInteractive(msalAuthTestParams2, new com.microsoft.identity.client.ui.automation.interaction.OnInteractionRequired() {
+            @Override
+            public void handleUserInteraction() {
+                final PromptHandlerParameters promptHandlerParameters = PromptHandlerParameters.builder()
+                        .prompt(PromptParameter.WHEN_REQUIRED)
+                        .loginHint(username)
+                        .sessionExpected(true)
+                        .consentPageExpected(false)
+                        .speedBumpExpected(false)
+                        .broker(mBroker)
+                        .expectingBrokerAccountChooserActivity(false)
+                        .passwordPageExpected(true)
+                        .build();
+
+                new AadPromptHandler(promptHandlerParameters)
+                        .handlePrompt(username, password);
+            }
+        }, TokenRequestTimeout.MEDIUM);
+
+        System.out.println("Completed acquireTokenInteractive post password change");
```

Comment:
I think there is already a logger being used by the test infra...why can't we use that here?

Reply:
removed this line was not necessary